src/t_stream.c

```diff
@@ -2205,9 +2205,9 @@ void xreadCommand(client *c) {
             streams_arg = i+1;
             streams_count = (c->argc-streams_arg);
             if ((streams_count % 2) != 0) {
-                addReplyError(c,"Unbalanced XREAD list of streams: "
-                                "for each stream key an ID or '$' must be "
-                                "specified.");
+                addReplyErrorFormat(c,"Unbalanced %s list of streams: "
+                                      "for each stream key an ID or '>' must be "
+                                      "specified.", (char *)c->argv[0]->ptr);
```

Comment:
maybe better use `c->cmd->realname` instead of the user input?

Reply:
yes, updated